ibis/backends/tests/conftest.py

```diff
@@ -306,3 +307,45 @@ def alternate_current_database(con, backend, current_data_db: str) -> str:
     finally:
         con.set_database(current_data_db)
         con.drop_database(name, force=True)
+
+
+@pytest.fixture
```

Comment:
are these *only* used in the new tests? if so i would just put them in the sql tests themselves for now. alt these can stay but give real names to these

Reply:
I was following the naming convention given for the other fixtures. Sure, I'll update the names.

Reply:
Done!